packages/ra-ui-materialui/src/form/SimpleForm.stories.tsx

```diff
@@ -145,6 +147,36 @@ export const GlobalValidation = () => (
     </Wrapper>
 );
 
+export const CustomNotificationMessage = () => (
```

Comment:
IMO this story is misplaced. The logic responsible for this feature is actually implemented in `useEditController`, not in `SimpleForm`.